src/i18n/en-events.ts

```diff
@@ -30,6 +30,7 @@ const eventsMessageDescriptors = {
   [`event.${ColonyAndExtensionsEvents.ColonyMetadata}.tokens`]: `{initiator} changed this colony's tokens`,
   [`event.${ColonyAndExtensionsEvents.ColonyMetadata}.verifiedAddresses`]: `{initiator} updated this colony's address book`,
   [`event.${ColonyAndExtensionsEvents.ColonyMetadata}.safeRemoved`]: `{initiator} removed {removedSafesString} using the Administration and Funding permissions.`,
+  [`event.${ColonyAndExtensionsEvents.ColonyMetadata}.safeAdded`]: `{initiator} added the Gnosis Safe {safeAddress} from {selectedChain} using the Administration and Funding permissions`,
```

Comment:
Same here - we might have a safe from a different chain so you need to add the chainName variable and pass it in (same as safeAddress).
@ArmandoGraterol, am I right?

Reply:
This might be a good question for @arrenv as from my understanding these safes may still be called "Gnosis Safe" even though they may not be in the "Gnosis Chain". 

For example, in `https://gnosis-safe.io` you can create safes from different chains other than the Gnosis chain, my suspicion is that "Gnosis Safe" may just be a "brand name" for the safes.

Reply:
I understand it that the product is called "Gnosis Safe", and the use of the term "safe" to refer to a multisig is specific to the Gnosis Safe platform. Therefore something like an "Aurora" safe doesn't exist. It would be a Gnosis Safe deployed on the Aurora network. 

@arrenv Is that right?

Reply:
@willm30 @ArmandoGraterol  Yes, "Gnosis Safe" is a brand. In fact they are rebranding it to just be "Safe" - https://www.cryptotimes.io/gnosis-safe-is-rebranded-to-safe-with-100m-fundraise

Yes, we need to be able to specify what chain the Safe is on.

Reply:
So, this would be: "{initiator} added the Safe {safeAddress} from {selectedChain} using the Administration and Funding permissions`"